City-of-Pori/pori-kada
# PORI-596: Businesspori branding and content page

- Add logo, Businesspori specific styles and colours, small styling tweaks.
- Adjust content page to "full width" and add a height restriction to main image as discussed in Jira ticket

Steps to test:
- Go through https://businesspori.dev.wunder.io/ and see that the site looks according to designs in regards of colours, logo
- Go to a content page with main image, e.g. https://businesspori.dev.wunder.io/porin-edut and see that the width is same as a content page without a main image https://businesspori.dev.wunder.io/clone-full-input-page-test-no-main-img
- See that the main image is restricted to 400px on desktop in content page.
